src/features/terminal/renderer/fonts/red-hat-mono.injectable.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { terminalFontInjectionToken } from "./token";
+import RedHatMono from "./RedHatMono-Regular.ttf";
+
+const redHatMonoTerminalFontInjectable = getInjectable({
+  id: "red-hat-mono-terminal-font",
+  instantiate: () => ({
+    name: "Red Hat Mono",
+    url: RedHatMono,
+  }),
+  injectionToken: terminalFontInjectionToken,
+});
+
+export default redHatMonoTerminalFontInjectable;
```

Comment:
_btw:_ why do we need to name some injectable when it's name never used in many files? we could do direct `default` export like this:

```ts
export default getInjectable({
  id: "red-hat-mono-terminal-font",
  instantiate: () => ({
    name: "Red Hat Mono",
    url: RedHatMono,
  }),
  injectionToken: terminalFontInjectionToken,
});
```

Reply:
In this case we could just do a `export default` since this will only ever be `di.injectMany`-ed. The reason to add the `const ...; export default ...;` is to help with intellisense and auto-import when using an injectable with `di.inject()`. This way it can easily be auto-imported if someone ever wants to import just this one injectable.